charts/actions-runner-controller/values.yaml

```diff
@@ -47,17 +47,15 @@ serviceAccount:
 podAnnotations: {}
 
 podSecurityContext:
-  {}
-  # fsGroup: 2000
+  fsGroup: 1000
 
 securityContext:
-  {}
   # capabilities:
   #   drop:
   #   - ALL
   # readOnlyRootFilesystem: true
   # runAsNonRoot: true
-  # runAsUser: 1000
+  runAsUser: 1000
```

Comment:
Does this break EKS IRSA integration? Are you able to still read the bootstrapped web identity token file? Which user is 1000 again?

Reply:
It's a user with non-root privileges, linux starts creating normal users at UID 1000. You are right though, this may not do much as distroless images are already being used and run as non-root.